doc/motion_planning_pipeline/CMakeLists.txt

```
@@ -1,8 +1,13 @@
 add_executable(motion_planning_pipeline_tutorial
     src/motion_planning_pipeline_tutorial.cpp)
-target_link_libraries(motion_planning_pipeline_tutorial
-    ${catkin_LIBRARIES} ${Boost_LIBRARIES})
-install(TARGETS motion_planning_pipeline_tutorial DESTINATION
-    ${CATKIN_PACKAGE_BIN_DESTINATION})
+target_include_directories(motion_planning_pipeline_tutorial
+    PUBLIC include)
```

Should it be like this?

```suggestion
target_include_directories(motion_planning_pipeline_tutorial
    PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
  "$<INSTALL_INTERFACE:include>"))
```

---

Hey @cmelliott2 thanks for the review! I think these two points to the same include directory, but I'll have to check it out and let you know.